util/test/checkChplInstall

```diff
@@ -96,6 +96,26 @@ else
     CHPL_DIR_REMOVE=0
 fi
 
+# Tmp directory for compile output
+TMP_TEST_DIR="$(mktemp -d ${CHPL_DIR}/chapel-test-XXXXX)"
+log_info "Temporary test job directory: ${TMP_TEST_DIR}"
+
+# Cleanup the tmp directory whenever exit is invoked
+function cleanup_tmp_dir()
+{
+    log_debug "Removing ${TMP_TEST_DIR}"
+    rm -rf ${TMP_TEST_DIR} || (
+        log_warning "failed Removing ${TMP_TEST_DIR}, ignore the error"
+        set -x; ls -la ${TMP_TEST_DIR}
+    ) || : ok
```

Comment:
$ type :
: is a shell builtin

: ok 
# is exactly the same as
true ok 
# "ok" is argv1, which is ignored

The reason there is such a thing as ":", is because when set -x is active, the shell echoes every command (including arguments after interpolation/expansion) right before it is executed. Thus you can use ":" plus arguments to write commentary which will appear in your set -x log in a very recognizable form; comments written with "#" will not appear at run time.